src/Microsoft.IdentityModel.Tokens/ECDsaSecurityKey.cs

```diff
@@ -116,7 +116,7 @@ public override int KeySize
         /// <remarks>https://tools.ietf.org/html/rfc7638</remarks>
         public override bool CanComputeJwkThumbprint()
         {
-#if NETSTANDARD2_0
+#if NET472 || NETSTANDARD2_0
```

Comment:
net472 supports [ECParameters](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.ecparameters?view=net-5.0#moniker-applies-to), there is no need for this check. Similar applies to ln 133.

You can also keep this logic as is, but refactor SupportsECParameters to return **true** on net472, without the unnecessary check.

Reply:
I've refactored SupportsECParameters to return true.